internal/system/agent/clients/agent.go

```diff
@@ -17,39 +17,39 @@ package clients
 import (
 	"sync"
 
-	"github.com/edgexfoundry/go-mod-core-contracts/clients/general"
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/agent"
 )
 
-// GeneralType is an alias for general.GeneralClient and hides implementation detail.
-type GeneralType general.GeneralClient
+// AgentType is an alias for general.GeneralClient and hides implementation detail.
```

Comment:
Does it make sense to:
s/general.GeneralClient/agent.AgentClient

Reply:
Yes, thank you.